src/plasoscaffolder.egg-info/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
```

Comment:
files in .egg-info are auto generated I suggest adding `/ src/plasoscaffolder.egg-info` to `.gitignore`.